### PR TITLE
Add two sum problem

### PR DESCRIPTION
**Pull request by arty-m**
- created class that calculates solution to two sum and returns a boolean
- created a main class that uses Two Sum and returns a boolean

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import TwoSum;
+
+public class Main {
```
Comment (9491013161) [inline Main.java]: please add documentation

File: TwoSum.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+public TwoSum {
+
+private int[] mNums;
+
```
Comment (9491013161) [inline TwoSum.java]: missing documentation

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	
+	public static void main(String[] args) {
```
Comment (9491013161) [inline Main.java]: missing documentation

File: TwoSum.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+private int mTarget;
+
+public TwoSum(int[] nums, int trgt) {
+	mNums = nums;
+	mTarget = trgt;
+}
+
```
Comment (9491013161) [inline TwoSum.java]: add documentation

**Comment by arty-m** [comment]
thank you for review. I forgot about documentation.

File: TwoSum.java (hunk 2 above)
Comment (9491013161) [inline TwoSum.java]: add  documentation

File: TwoSum.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+public boolean result() {
+	try {
+		int[] nums = twoSum();
+		return true;
+	} catch (IllegalArgumentException e) {
+		return false;
+	}
+	return false;
+}
+
```
Comment (9491013161) [inline TwoSum.java]: Please Explain this Method

**Comment by arty-m** [comment]
Please re-review.

**Pull request merged**